requirements-dev.txt

```diff
@@ -26,7 +26,7 @@ mypy>=1.5.0
 ninja
 torchvision
 psutil
-cucim>=23.2.0; platform_system == "Linux"
+cucim-cu12; platform_system == "Linux" and python_version >= "3.9" and python_version <= "3.10"
```

Comment:
This is related to cucim compatibility issues or specifically related to ARM?

Reply:
There are two key reasons for the update:
- Firstly, as highlighted by @drbeh, cucim will no longer be supporting PyPI in the future. Hence, we need to transition to a package with a different name.
- Secondly, this new package provides support for ARM architecture, unlike the original one.